spec/outputs/webhdfs_spec.rb

```diff
@@ -0,0 +1,142 @@
+# encoding: utf-8
```

Comment:
Done in https://github.com/dstore-dbap/logstash-output-webhdfs-1/commit/043b677fe328de98958686b69d6d0a80c149b762